src/agent/onefuzz-agent/src/local/common.rs

```diff
@@ -32,6 +32,7 @@ pub const CHECK_ASAN_LOG: &str = "check_asan_log";
 pub const TOOLS_DIR: &str = "tools_dir";
 pub const RENAME_OUTPUT: &str = "rename_output";
 pub const CHECK_FUZZER_HELP: &str = "check_fuzzer_help";
+pub const DISABLE_CHECK_DEBUGGER: &str = "disable_check_debugger";
```

Comment:
WDYT about calling this `no_check_debugger`? Yours is nicer English, but the `no-$OPTION` pattern feels more familiar in a CLI arg context.

Reply:
Oh, nevermind. You were just pulling an existing magic string into a constant. Carry on.